src/vinyldns/client.py

```diff
@@ -589,14 +589,21 @@ def list_record_set_changes(self, zone_id, start_from=None, max_items=None, **kw
         response, data = self.__make_request(url, u'GET', self.headers, **kwargs)
         return ListRecordSetChangesResponse.from_dict(data)
 
-    def create_batch_change(self, batch_change_input, **kwargs):
+    def create_batch_change(self, batch_change_input, allow_manual_review=None, **kwargs):
         """
         Create a new batch change.
 
         :param batch_change_input: the batchchange to be created
+        :param allow_manual_review: set to false to fail rather than go to
```

Comment:
Maybe `optional override to opt out of manual review` is clearer for the description.

Reply:
say that 5 times fast